lib/task-data/tasks/catalog-smart.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
         commands: [
             {
                 command: 'sudo bash get_smart.sh',
-                downloadUrl: '/api/current/templates/get_smart.sh'
+                downloadUrl: '{{api.templates}}/get_smart.sh?nodeId={{context.target}}'
```

Comment:
This doesn't work for task rendering, but only for templates and profiles rendering.
Thanks.